neo/rawio/blackrockrawio.py

```diff
@@ -645,8 +646,13 @@ def _get_spike_raw_waveforms(self, block_index, seg_index, unit_index, t_start,
 
         wf_dtype = self.__nev_params('waveform_dtypes')[channel_id]
         wf_size = self.__nev_params('waveform_size')[channel_id]
+        wf_byte_size = np.dtype(wf_dtype).itemsize * wf_size
 
-        waveforms = unit_spikes['waveform'].flatten().view(wf_dtype)
+        dt1 = [
+            ('extra', 'S{}'.format(unit_spikes['waveform'].dtype.itemsize - wf_byte_size)),
+            ('ch_waveform', 'S{}'.format(wf_byte_size))]
```

Comment:
How did you get this specification of extra bytes ? is it in the official documents ?

Reply:
Not really, the code looked like it follows the documentation, but I saw that the available bytes for the waveforms are bigger that the ones expected given the dtype.  Maybe the reserved bytes ( the ones before the waveform data) must be dynamic and not always uint8 (but I couldn't find documentation to support that theory). For that reason I did that dt1 type, just to extract the expected bytes

Reply:
But how we can be sure that extra bytes are before ?
And what if extra size == 0 ?

Reply:
good question, I don't know, I just tested it in a file and there is the relationship with the reserved bytes that are before. (I can share the file if you want)
I checked the case when extra size == 0 and it's fine, it just creates an empty binary field without errors